internal/operators/ocs/ocs_operator.go

```diff
@@ -77,20 +77,18 @@ func (o *operator) ValidateCluster(_ context.Context, cluster *common.Cluster) (
 func (o *operator) ValidateHost(ctx context.Context, cluster *common.Cluster, host *models.Host) (api.ValidationResult, error) {
 	numOfHosts := len(cluster.Hosts)
 	if host.Inventory == "" {
-		o.log.Info("Empty Inventory of host with hostID ", host.ID)
 		return api.ValidationResult{Status: api.Pending, ValidationId: o.GetHostValidationID(), Reasons: []string{"Missing Inventory in some of the hosts"}}, nil
 	}
 	inventory, err := hostutil.UnmarshalInventory(host.Inventory)
 	if err != nil {
-		o.log.Errorf("Failed to get inventory from host with id %s", host.ID)
-		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID()}, err
+		message:=fmt.Sprintf("Failed to get inventory from host with id %s", host.ID)
```

Comment:
Nit: formatting. Please leave a space before and after ":="
Same comment holds elsewhere.

Reply:
done